monkey/monkey_island/cc/ui/src/components/report-components/zerotrust/scoutsuite/ScoutSuiteRuleButton.js

```diff
@@ -27,18 +27,27 @@ export default class ScoutSuiteRuleButton extends Component {
                              hideCallback={this.toggleModal} />
         <div className="text-center" style={{'display': 'grid'}}>
           <Button variant={'monkey-info'} size={'lg'} onClick={this.toggleModal}>
-            <FontAwesomeIcon icon={faList}/> ScoutSuite rules {this.createRuleCountBadge()}
+            <FontAwesomeIcon icon={faList}/> ScoutSuite rules
+            &nbsp;<RuleCountBadge count={this.props.scoutsuite_rules.length}/>
           </Button>
         </div>
       </>);
   }
 
   createRuleCountBadge() {
-    const ruleCount = this.props.scoutsuite_rules.length > 9 ? '9+' : this.props.scoutsuite_rules.length;
-    return <Badge variant={'monkey-info-light'}>{ruleCount}</Badge>;
+
   }
 }
 
+function RuleCountBadge(props) {
+  const maxRuleCountToShow = 9;
```

Comment:
The first 2 are consts in a global manner. Move them out of the function, and use UPPER CASE NAMES for them

Reply:
What do you mean by "global manner"? These are consts related only to `RuleCountBadge` component.

Reply:
These consts are the same every time `RuleCountBadge` is called, that's why it doesn't make sense to me to declare them inside the function.

I understand this function represents a component, and I'm not up-to-date with the latest javascript practices, so I might be wrong on this one.
The way I see it, this `RuleCountBadge` component can either be extracted to a different file (which is probably overkill here) or can stay in the same file, but I don't see a reason to assign these consts inside the function

Reply:
In React components are functions. You're right, this should be exported to a separate component and reused